### DISCUSSION TITLE
What was used to create Darth Vader's iconic breathing sound?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#726] [Galaxy Trivia] [answered]
*answer without using words* 🙃

**Comment by SANKARAMDAS** [comment]
![darth-vader](https://github.com/ParanoidUser/yolo/assets/31897843/89282c13-f2d3-440c-94be-7d7e7d609543)

**Comment by peterbaccaro** [comment, marked as answer]
![DACOR](https://github.com/ParanoidUser/yolo/assets/74666079/1a142ed9-16e7-48c7-8b81-ba15806fac24)

An old Dacor scuba regulator was used for Darth Vader's ominous heavy breathing in the Star Wars films.
